src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splitString = sourceString.trim().split(';');
+
+  splitString.forEach((item) => {
+    const splitItem = item.trim().split(':');
+
+    if (splitItem.length === 2) {
+      const key = splitItem[0].trim();
+      const value = splitItem[1].trim();
+
+      result[key] = value;
+    }
```

Comment:
it looks weird)
ur if can look like this
```suggestion
    if (key && value) {
      result[key.trim()] = value.trim();
    }
```